recipes/ntp/ntp.inc

```diff
@@ -45,6 +45,14 @@ do_install_extra_files() {
     install -m 755 ${SRCDIR}/ntpd ${D}/${sysconfdir}/init.d/ntpd
 }
 
+RECIPE_FLAGS +="ntpd_start_args"
```

Comment:
Most likely never.  But I could very well want to have a single distro that supports two different ntpds.  Actually, I do that now, so I am pretty sure it is a realistic scenario :)